projects/components/src/input/input.component.ts

```diff
@@ -71,6 +72,9 @@ export class InputComponent<T extends string | number> implements ControlValueAc
   @Output()
   public readonly inputFocus: EventEmitter<void> = new EventEmitter();
 
+  @Output()
+  public readonly inputBlur: EventEmitter<void> = new EventEmitter();
```

Comment:
Could you clarify why we'd need to manually propagate blur rather than use focusin/focusout?